### ISSUE TITLE
Seqfault in WX backend following the merge of Phoenix 

### ISSUE THREAD

**Issue by jenshnielsen** [matplotlib/matplotlib#4461]
I am seeing a Seqfault with the WX backend  using WX Classing (3.03 from homebrew on OSX 10.10)
Following the merge of #3421 

Sorry I should have tested this earlier but did not get round to it. 

Doing 

```python 
import matplotlib
matplotlib.use('WXAgg')
import matplotlib.pyplot as plt
plt.plot(range(100))
plt.show()
```

Results in a seqfault when run with python 2.7.9 and WX 3.03 classic

https://gist.github.com/jenshnielsen/0106d0b963ba50268824

**Comment by WeatherGod** [comment]
attn: @wernerfb

**Comment by wernerfb** [comment]
Hhm, can't test on Mac, but will look into this on Windows.  Never used mpl.pyplot so might take a bit of time.

**Comment by jenshnielsen** [comment]
closed by #4479
